components/email/src/Email.svelte

```diff
@@ -773,6 +773,46 @@
     return fromString;
   }
 
+  function getParticipants(thread: Thread): string[] {
+    const messages = thread.messages;
+    const drafts = thread.drafts;
+    const lastMessageFrom = messages[messages.length - 1]?.from;
+    const hasParticipant =
+      messages && participants && messages.length > 0 && lastMessageFrom.length;
+
+    let participantsList = [];
+    if (drafts.length) {
+      participantsList.push("Draft");
+    }
+    if (hasParticipant) {
+      participantsList.push(
+        lastMessageFrom[0].name || lastMessageFrom[0].email,
+      );
+
+      //Only display 2 participants
+      const secondParticipant = messages
+        .slice(0, -1)
+        .reverse()
+        .find((msg) => {
+          return (
+            msg.from?.length > 0 &&
+            msg.from[0].email !== lastMessageFrom[0].email
+          );
+        });
+
+      if (secondParticipant) {
+        const participantString =
+          secondParticipant.from[0].name || secondParticipant.from[0].email;
+        participantsList.push(participantString);
+      }
+    }
+    return participantsList;
+  }
+
```

Comment:
Reactive boolean `isDeleted` is extracted here, used to determine if its a deleted email.